Representable/representable
# map page: search while moving the map + display # of COIs

**changes**
- on map page, displays the number of communities at bottom of sidebar
- on map page, adds button "Search while moving the map" -- when toggled, only displays the currently visible communities on the sidebar (and updates community number displayed)

**to test**
- python manage.py runserver
- go to state map page
- scroll around and see that behavior is normal
- click "Search while moving the map" button
- see that only visible communities show on sidebar, and that number of COIs is updated
- de-select button
- see that all communities re-appear on sidebar, and that dragging the map does not alter what's visible or the number at the bottom of the list
- test all the same for a drive/org map page

**screenshots**
comm number: 
![image](https://user-images.githubusercontent.com/41943646/125537132-85c663be-d979-48b6-85d2-27ed00eedd77.png)
search button (inactive):
![image](https://user-images.githubusercontent.com/41943646/125537151-93ed80b9-52d4-4b25-b87e-a9623180b63c.png)
search button (active):
![image](https://user-images.githubusercontent.com/41943646/125537167-7a557488-939a-4932-8951-7f4e5c8156e7.png)
